main.yml

```diff
@@ -233,14 +233,27 @@ cost-management:
         title: Overview
         default: true
         group: cost-management
-      - id: details
+      - id: ocp
+        title: OpenShift
+        group: cost-management
+      - id: infrastructure
         group: cost-management
       - id: cost-models
         title: Cost models
         group: cost-management
   source_repo: https://github.com/project-koku/
   mailing_list: cost-mgmt@redhat.com
 
+infrastructure:
+  title: Infrastructure
+  frontend:
+    sub_apps:
+      - id: aws
+        title: Amazon Web Services
+        default: true
+      - id: azure
+        title: Microsoft Azure
+
 details:
```

Comment:
Since you're removing the "details" reference, please also remove this "details" config.

Reply:
Yes, pushed to remove the old details. Thank you.